<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>products</class>
 <widget class="QWidget" name="products">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="type" stdset="0">
   <string>main</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="type" stdset="0">
      <string>mainframe</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1" alignment="Qt::AlignTop">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Add New Brand</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Brand Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="fieldNewBrand">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0" alignment="Qt::AlignRight">
          <widget class="QPushButton" name="buttonAddBrand">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignLeft">
       <widget class="QTableWidget" name="tableBrands">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Brand Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Brand Name</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_5">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="buttonDeleteBrand">
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUpdateBrand">
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_6">
     <property name="type" stdset="0">
      <string>mainframe</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="1" alignment="Qt::AlignTop">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Add New Category</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Category Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="fieldNewCategory">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0" alignment="Qt::AlignRight">
          <widget class="QPushButton" name="buttonAddCategory">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignLeft">
       <widget class="QTableWidget" name="tableCategories">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Category Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Category Name</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_7">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="buttonDeleteCategory">
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUpdateCategory">
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="type" stdset="0">
      <string>mainframe</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0" alignment="Qt::AlignRight">
       <widget class="QFrame" name="frame_3">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="buttonDeleteProduct">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUpdateProduct">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Add New Product</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Current Stock</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Product Name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Brand</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="fieldName">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="fieldStock">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="fieldCategory">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Category</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="fieldBrand">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="fieldPrize">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Prize/Unit</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="6">
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QPushButton" name="buttonAddProduct">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="buttonClear">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QTableWidget" name="tableProducts">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Product Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Product Name</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
